charts/argo-workflows/templates/server/server-deployment.yaml

```diff
@@ -12,7 +12,9 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  {{- if (ne .Values.server.autoscaling.enabled true) }}
   replicas: {{ .Values.server.replicas }}
+  {{- end }}
```

Comment:
`replicas` is not supposed to be defined when autoscaling is enabled, maybe condition flag is opposite? 🤔 
```diff
-{{- if (ne .Values.server.autoscaling.enabled true) }}
+{{- if not .Values.server.autoscaling.enabled }}
  replicas: {{ .Values.server.replicas }}
{{- end }}
```

Reply:
exiting logic is https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/argocd-server/deployment.yaml#L19-L21 .

Reply:
Ah good catch! The logic still works ("not equal true"), but yea the newer version is cleaner/simpler than the older. I'll update that

Reply:
updated / fixed